experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1483,15 +1483,23 @@ def _validate_versions(self, data):
             )
 
         if (
-            application == NimbusExperiment.Application.DESKTOP
+            (
+                application != NimbusExperiment.Application.KLAR_ANDROID
+                and application != NimbusExperiment.Application.KLAR_IOS
+            )
             and is_rollout
             and parsed_min_version
             < NimbusExperiment.Version.parse(
-                NimbusConstants.DESKTOP_ROLLOUT_MIN_SUPPORTED_VERSION
+                NimbusConstants.ROLLOUT_MIN_SUPPORTED_VERSION[application]
             )
         ):
             self.warnings["firefox_min_version"] = [
-                NimbusConstants.ERROR_DESKTOP_ROLLOUT_VERSION
+                NimbusConstants.ERROR_ROLLOUT_VERSION.format(
+                    application=NimbusExperiment.Application(application).label,
+                    version=NimbusExperiment.Version.parse(
+                        NimbusConstants.ROLLOUT_MIN_SUPPORTED_VERSION[application]
+                    ),
```

Comment:
Thinking I may refactor the parsed min-supported version out into a variable too since it's used twice now